src/main/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolver.scala

```diff
@@ -29,14 +31,14 @@ object MethodConfigResolver {
    * @return map from input name to a Try containing a resolved value or failure
    */
   def resolveInputs(workspaceContext: WorkspaceContext, methodConfig: MethodConfiguration, entity: Entity, wdl: String, txn: RawlsTransaction): Map[String, Try[Attribute]] = {
-    val wdlInputs = WdlNamespace.load(wdl).workflows.head.inputs
+    val wdlInputs = NamespaceWithWorkflow.load(wdl, BackendType.LOCAL).workflow.inputs
     txn withGraph { graph =>
       val evaluator = new ExpressionEvaluator(new ExpressionParser)
       methodConfig.inputs.map {
         case (name, expression) => {
           val evaluated = evaluator.evalFinalAttribute(workspaceContext, entity.entityType, entity.name, expression.value)
           // now look at expected WDL type to see if we should unpack a single value from the Seq[Any]
-          val unpacked = evaluated.flatMap(sequence => wdlInputs.get(name) match {
+          val unpacked = evaluated.flatMap(sequence => wdlInputs.find((w:WorkflowInput) => name == w.fqn) match {
```

Comment:
Won't this be matching on the ```WorkflowInput ``` rather than the ```WdlType```? Also could you do ```wdlInputs.map(_.fqn -> _.wdlType).toMap``` at the beginning to avoid the ```find```?